kernel/nfft/nfft.c

```diff
@@ -2478,6 +2478,12 @@ static void nfft_adjoint_1d_B(X(plan) *ths)
 
 void X(trafo_1d)(X(plan) *ths)
 {
+  if((ths->N[0] <= ths->m) || (ths->n[0] <= 2*ths->m+2))
```

Comment:
Ok, let's move forward on this one, but we should see if this limitations can be removed or not. At least the documentation would need to be updated if the code has to remain as it is. Maybe @skunis or @DanielPotts can help.